force-app/main/default/classes/HandlerBeforeAfterConcept.cls

```diff
@@ -0,0 +1,93 @@
+/**  @classname  			:   HandlerBeforeAfterConcept
+  *  @author 				:   Raghav
+  *  @date 					:   01-02-2023
+  *  @description  			:   This is Handler class of TriggerAllEventConcept used to check after and before concepts of trigger context variables
+  *  @modification log		:	
+  *  1. Comments added      :   01-02-2023
+*/
+public with sharing class HandlerBeforeAfterConcept {
+/**  @methodname 	    :   beforeInsert 
+  *  @author 			:   Raghav
+  *  @group  			:   none
+  *  @param  			:   List of Account
+  *  @description       :   This method print wheter trigger context varible worked or not accordingly
+  *  @return 			:   none
+*/
+  public void beforeInsert(List<Account>accList){
+    for (Account acc : accList) {
+      System.debug('Before Insert=='+acc);
```

Comment:
There are some special logging level define by Salesforce you can use any specific of them to perform debug logging. Try to avoid plain ```apex System.debug(); ``` method
Some of the logging levels like below
```java
System.logginglevel.INFO
System.logginglevel.ERROR
System.logginglevel.DEBUG
System.logginglevel.WARN
System.logginglevel.FINE
System.logginglevel.FINER
System.logginglevel.FINEST
```